### ISSUE TITLE
Create documentation

### ISSUE THREAD

**Issue by MilanDierick** [Parexy/Multiplayer#7]
Currently we don't have any documentation of the code, which will make it pretty difficult to get anything done in a timely fashion. Furthermore, some documentation will allow us to identify what we need to implement ourselves and what has already been implemented.

We also need to identify which systems currently exist, and how they work together. By doing this, we most likely will be able to identify syncing problems and systems we can improve to make development more efficient.

**Comment by tindolt-zz** [comment]
would you like help?

**Comment by Dunkhan** [comment]
Yeah I already started on this so feel free to copy what I have so far: https://github.com/Dunkhan/Multiplayer/wiki/Guide-to-making-UI-buttons-compatible.

Edit actually, the API will make all of the UI buttons stuff obsolete wont it.

Well I was planning to write a debugging page next, I learned  a few tricks there.

**Comment by tindolt-zz** [comment]
@Dunkhan we plan on hopefully making it easier and a lot of people dont like using the api, its nice to have mods not rely on yet another mod

**Comment by tindolt-zz** [comment]
This is put in wiki under documentation, can we close this issue and continue on the wiki one?
